keyboards/albert_smart/albert_smart84/rules.mk

```diff
@@ -0,0 +1,14 @@
+MCU_LDSCRIPT = STM32F103xB
+
+DEFERRED_EXEC_ENABLE = yes
```

Comment:
should be moved to features, in info.json (eg, `"deferred_exec": "true"`

Reply:
ok,I add it to info.json